packages/react-components/src/components/TextFieldWithMention/mentionTagUtils.ts

```diff
@@ -905,24 +906,45 @@ const parseOpenTag = (tag: string, startIdx: number): TagData => {
   };
 };
 
-const findNextHtmlTag = (text: string, startIndex: number): HtmlTag | undefined => {
-  const tagStartIndex = text.indexOf('<', startIndex);
-  if (tagStartIndex === -1) {
-    // No more tags
-    return undefined;
+const findNextHtmlTag = (text: string, startIndex: number, tagType = MSFT_MENTION_TAG): HtmlTag | undefined => {
+  const newText = text.substring(startIndex);
+  let type: HtmlTagType = 'open';
+  let regexResult;
+
+  const openTagRegex = new RegExp(`<${tagType}(.*?)>`, 'i');
```

Comment:
How much of a difference in performance is there between the regex and the previous implementation?